spacelift/resource_aws_role.go

```diff
@@ -218,7 +224,7 @@ func resourceAWSRoleSet(ctx context.Context, client *internal.Client, id string,
 	var mutation struct {
 		AttachAWSRole struct {
 			Activated bool `graphql:"activated"`
-		} `graphql:"stackIntegrationAwsCreate(id: $id, roleArn: $roleArn, generateCredentialsInWorker: $generateCredentialsInWorker, externalID: $externalID)"`
+		} `graphql:"stackIntegrationAwsCreate(id: $id, roleArn: $roleArn, generateCredentialsInWorker: $generateCredentialsInWorker, externalID: $externalID, durationSeconds: $duration_seconds)"`
```

Comment:
nit: `$durationSeconds` instead of `$duration_seconds`

Reply:
+1